src/citrine/gemtables/columns.py

```diff
@@ -283,7 +283,7 @@ def _attrs(self) -> List[str]:
     def __init__(self, *,
                  data_source: str,
                  component_name: str,
-                 normalize: Optional[bool] = False):
+                 normalize: bool = False):
```

Comment:
This is technically a breaking change, since if someone has a script in which they define a `ComponentQuantityColumn` and set `normalize=None`, it will no longer serialize. But I strongly doubt anyone has done that.

Reply:
Why are we getting rid of the Optional?  It matches the definition in [backend](https://github.com/CitrineInformatics/platform-backend/blob/590ea23c052ae2600aab43e39c7ea9edd4c49fdd/code/gemtables/model/src/main/scala/io/citrine/gemtables/model/column/CompositionColumnDefinition.scala#L126) and are keeping the default.  Am I missing the purpose of the Optional typehint?

Reply:
Optional means you can provide `None` and in doing so convey information. What does `None` mean in this context? Either you want to normalize or you don't. It's true that, strictly speaking, you can provide `None` and the backend fills in the default value of `False`. But that's not how `Optional` is generally used in the python client. For example, in listing calls, the `page` is optional. Not specifying a page means something different from specifying a page.

The definition on the backend isn't optional either. It's `boolean`, not `Some[boolean]`.  Having a default is not the same as being optional.

Reply:
Ahh, yes, that tracks.  We don't need ternary logic.